ApollosProject/apollos-apps
# chore: remove the Community slide from the LandingSwiper

## ✏️ Description



This PR removes the Community Slide from the Landing Swiper, as this feature is not used in any production apps yet. Additional context: We need to be able to disable these screens if a church does not have that feature in their app. We also need to build a default Explore Content slide to add value to this swiper (when a church disables Community Slide, for instance)

This PR lays the ground work for that.

## 🔬 To Test
1. Pull down this code
2. Check to see if you have 3 slides on the Landing Swiper
3. Check to see that the slides progress as expected to the Auth screen

## 📸 Screenshots

|Before|After|
|---|---|
|![Simulator Screen Shot - iPhone 14 Pro - 2022-12-05 at 14 21 43](https://user-images.githubusercontent.com/72768221/205735973-890a0239-0ca4-4b15-93e5-aeb55d1dcd9a.png)|![Simulator Screen Shot - iPhone 14 Pro - 2022-12-05 at 14 21 17](https://user-images.githubusercontent.com/72768221/205735976-694dfa7c-255c-4e5e-ae5a-7080c7739992.png)|

## ⚠️ To-do Before Merge




## 📝 Checklist
